### PR TITLE
Improve readme on rfcs repository

### PR DESCRIPTION
**Pull request by laurakalbag**
This isn’t an RFC itself 😂

I’ve make some small tweaks to the RFC template (mostly formatting) and written a readme inspired by Svelte and React’s RFC readmes.

Feedback on the process itself or the readme both welcome!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,47 @@
...
+
+Our RFC process is inspired by the process adopted by [Svelte](https://github.com/sveltejs/rfcs), [React](https://github.com/reactjs/rfcs), [Ember](https://github.com/emberjs/rfcs) and others. The process itself is subject to change, or may even be abandoned, as we gain experience.
+
+
+## The process
+
+1. Fork the RFC repo at http://github.com/statelyai/rfcs
+2. Copy 0000-template.md to text/0000-my-feature.md, where ‘my-feature’ is descriptive. Don’t assign an RFC number yet.
```
Comment (davidkpiano) [inline README.md:19]: Nit: I like putting filenames in code blocks for easier readability; happy to hear your opinion on this though

```suggestion
2. Copy `0000-template.md` to `text/0000-my-feature.md`, where ‘my-feature’ is descriptive. Don’t assign an RFC number yet.
```

File: README.md [hunk 2]
```diff
@@ -1,2 +1,47 @@
...
+
+However, getting merged is not the only success criteria for an RFC. Even when the design does not match the direction we’d like to take, we find RFC discussions very valuable for research and inspiration. Whenever we start work on a related area, we check the RFCs in that area and review the use cases and concerns the community members have posted. When you send an RFC, your primary goal should not necessarily be to get it merged into XState or the Stately tools as is, but to generate a valuable discussion with the community members. If your proposal later becomes accepted, that’s great. But even if it doesn’t, your work isn’t for nothing! The resulting discussion often informs the next proposal in the same problem space, whether the proposal comes from the community or the team.
+
+Our RFC process is inspired by the process adopted by [Svelte](https://github.com/sveltejs/rfcs), [React](https://github.com/reactjs/rfcs), [Ember](https://github.com/emberjs/rfcs) and others. The process itself is subject to change, or may even be abandoned, as we gain experience.
+
+
+## The process
```
[the remark this answers is not in the document]
Comment (laurakalbag) [reply README.md:16]: @davidkpiano please help me make this better! https://stately.ai/registry/editor/share/eee673df-9f4d-4965-beb8-1a649676521a

[the remark this answers is not in the document]
Comment (davidkpiano) [reply README.md:16]: I think it looks great as-is!

[the remark this answers is not in the document]
Comment (laurakalbag) [reply README.md:16]: rfc-statechart

File: README.md [hunk 3]
```diff
@@ -1,2 +1,49 @@
...
+
+![Statechart representing the RFC process detailed below.](https://user-images.githubusercontent.com/266663/169092679-3f64947d-31e6-4e04-bdba-baf408dfd832.png) [View this statechart in the Stately editor](https://stately.ai/registry/editor/share/eee673df-9f4d-4965-beb8-1a649676521a)
```
Comment (Andarist) [inline README.md:18]: nice Studio flex 💪 :P 

soon we should also be able to include a light-mode picture like described here: https://github.blog/changelog/2021-11-24-specify-theme-context-for-images-in-markdown/

**Review by Andarist — APPROVED** [review]
Really stately document!

File: README.md [hunk 4]
```diff
@@ -47,3 +47,15 @@ If you are interested in working on the implementation for an “active” RFC,
...
+
+## Commenting on RFCs
+
+Good feedback is an essential part of the RFC process. The RFC author has put a lot of time and effort into their design and should expect constructive feedback that helps improve their RFC. We absolutely welcome positive feedback, and if you don’t have a comment that contributes to the RFC’s improvement, we encourage you to show your support using a positive emoji (this also ensures the author doesn’t get overwhelmed by notifications!)
+
+Helpful feedback includes:
+- Your real-life use cases that are related to the RFC. Would the RFC solve your use case? Is your use case close to the RFC but would remain unsolved by the proposed design?
+- Alternative designs or solutions that solve the same problems described in the RFC. If you have an alternative that may improve upon the RFC’s design, explain your reasoning while respecting the author’s chosen approach.
+=======
+>>>>>>> a0a5724478259a3ab1197c8a1700bcbb4dfcc0b1
```
Comment (davidkpiano) [inline README.md:61]:
```suggestion

## Commenting on RFCs

Good feedback is an essential part of the RFC process. The RFC author has put a lot of time and effort into their design and should expect constructive feedback that helps improve their RFC. We absolutely welcome positive feedback, and if you don’t have a comment that contributes to the RFC’s improvement, we encourage you to show your support using a positive emoji (this also ensures the author doesn’t get overwhelmed by notifications!)

Helpful feedback includes:
- Your real-life use cases that are related to the RFC. Would the RFC solve your use case? Is your use case close to the RFC but would remain unsolved by the proposed design?
- Alternative designs or solutions that solve the same problems described in the RFC. If you have an alternative that may improve upon the RFC’s design, explain your reasoning while respecting the author’s chosen approach.
```

**Pull request merged**